scrapy/core/downloader/handlers/http11.py

```diff
@@ -319,9 +319,13 @@ def _get_agent(self, request, timeout):
                     pool=self._pool,
                 )
             else:
+                proxyScheme = b'http' if not proxyScheme else proxyScheme
+                proxyHost = to_bytes(proxyHost, encoding='ascii')
+                proxyPort = to_bytes(str(proxyPort), encoding='ascii')
+                proxyURI = urlunparse((proxyScheme, proxyNetloc, proxyParams, '', '', ''))
                 return self._ProxyAgent(
                     reactor=reactor,
-                    proxyURI=to_bytes(proxy, encoding='ascii'),
```

Comment:
Had to construct `proxyURI` and pass it to `_ProxyAgent` since `proxy` is the schemaless url with the incorrect format (not starting with `//`)

Reply:
If it works, :100: